packages/babel-plugin-polyfill-corejs3/src/built-in-definitions.js

```diff
@@ -457,6 +521,7 @@ export const StaticProperties: ObjectMap<
       "es.object.to-string",
       "es.math.to-string-tag",
       "es.json.to-string-tag",
+      "es.reflect.to-string-tag",
```

Comment:
It's better to add it on global `Reflect`, with `es.object.to-string` dependency.

Reply:
We don't have a global `JSON` builtin depending on `es.json.to-string-tag`. What's the difference here?

Reply:
Even by ES5, `JSON + '' === '[object JSON]'`, but till the latest ES version `Reflect + '' === '[object Object]'`, not `[object Reflect]`.

Reply:
It's better to remove it from `Symbol.toStringTag` deps.